app/src/main/java/com/example/riden/fragments/ProfileFragment.java

```diff
@@ -1,66 +1,85 @@
 package com.example.riden.fragments;
 
+import android.content.Intent;
 import android.os.Bundle;
 
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
 
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.ImageButton;
+import android.widget.ImageView;
+import android.widget.TextView;
 
+import com.bumptech.glide.Glide;
 import com.example.riden.R;
+import com.example.riden.models.User;
+import com.parse.GetCallback;
+import com.parse.ParseException;
+import com.parse.ParseFile;
+import com.parse.ParseObject;
 
-/**
- * A simple {@link Fragment} subclass.
- * Use the {@link ProfileFragment#newInstance} factory method to
- * create an instance of this fragment.
- */
 public class ProfileFragment extends Fragment {
+    ImageButton ibProfileImage;
+    TextView tvUsername;
+    TextView tvFullName;
+    TextView tvCar;
+    ImageView ivCarImage;
 
-    // TODO: Rename parameter arguments, choose names that match
-    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
-    private static final String ARG_PARAM1 = "param1";
-    private static final String ARG_PARAM2 = "param2";
-
-    // TODO: Rename and change types of parameters
-    private String mParam1;
-    private String mParam2;
-
+    public User user = (User) User.getCurrentUser();
```

Comment:
Should we override the ParseUser.getCurrentUser() function in User model to do this conversion (User) in there so that it is clean in the consumed classes ?